src/ui/GlobalStyle.js

```diff
@@ -224,6 +222,9 @@ class Utils extends StyleSheet {
     }
 }
 
+// TODO simplify this
+const GlobalStyle = Utils.getInstance();
```

Comment:
This seems pretty shady to me, maybe because of the naming. Also, we save that instance in GlobalStyle.Utils too ???